design/CreatePerformanceProfile.md

```diff
@@ -5,30 +5,42 @@ tuned.
 
 # Attributes
 
-* name \
+- **name** \
```

Comment:
Rename this file to `PerformanceProfile.md`. Add a link to the API associated with PerformanceProfiles at the end and rename that file back to `PerformanceProfileAPI.md`